pythran/config.py

```diff
@@ -170,9 +228,22 @@ def run():
 
     extension = pythran.config.make_extension(python=args.python)
 
+    if args.verbose >= 1:
+        logger.setLevel(max(logging.DEBUG, logging.ERROR - args.verbose * 10))
+        lint_cfg(cfg)
+        args.compiler = True
+        args.cflags = True
+        args.libs = True
+
+    if args.verbose >= 2:
+        output.append("CXX =".rjust(10))
```

Comment:
`output` does to stdout. that's not waht you want, if I'm not mistaken; Why not just log this info?

Reply:
It is what I intended, a sort of enhancement for the `pythran-config -v` behaves like a shortcut intead of `pythran-config --compiler --cflags --libs`. And `pythran-config -vv` makes the same output more readable as follows:

```sh
❯❯❯ pythran-config --no-python -vv > script.sh
❯❯❯ cat script.sh                                                  (pythran) enh/pythranrc-sanity-check ◼
     CXX = clang++ 
  CFLAGS = -D__PYTHRAN__=3 -DPYTHRAN_BLAS_OPENBLAS -I/home/avmo/src/pythran/pythran -I/home/avmo/src/pythran/pythran/pythonic/patch 

```

Which is more readable and can be output to a file to be easily modified to behave like a bash script if needed. 

If you don't like it I can revert it.